recipes/libsafec/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+import glob
+import os
+
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibSafeCConan(ConanFile):
+    name = "libsafec"
+    description = "This library implements the secure C11 Annex K[^5] functions" \
+            " on top of most libc implementations, which are missing from them."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rurban/safeclib"
+    license = "MIT"
+    topics = ("conan", "safec", "libc")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    build_requires = "autoconf/2.69", "libtool/2.4.6"
+    exports_sources = "patches/*"
+
+    __autotools = None
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _supported_compiler(self):
+        compiler = self.settings.compiler
+        version = tools.Version(self.settings.compiler.version)
+        if compiler == "Visual Studio":
+            return False
+        if compiler == "gcc" and version < "5":
+            return False
+        return True
+
+    def configure(self):
+        if not self._supported_compiler:
+            raise ConanInvalidConfiguration(
+                "libsafec doesn't support {}/{}".format(
+                    self.settings.compiler, self.settings.compiler.version))
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("safeclib-*")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    @property
+    def _autotools(self):
+        if self.__autotools:
+            return self.__autotools
+        self.run("autoreconf -fiv", run_environment=True)
```

Comment:
Move this to `build`.
Because this will also be run when doing a build with `-kb`.
```
conan create libsafec/3.6.0@
conan create libsafec/3.6.0@ -kb
```

Reply:
Make sense! 
Done: https://github.com/conan-io/conan-center-index/pull/4434/commits/6ef55741ca3870318d767de93711aa21d011c871